model/host/host.go

```diff
@@ -739,3 +744,53 @@ func CountInactiveHostsByProvider() ([]InactiveHostCounts, error) {
 	}
 	return counts, nil
 }
+
+// find all containers
+func (h *Host) FindAllContainers() ([]Host, error) {
+	query := db.Query(bson.M{
+		ContainerID: bson.M{"$exists": true},
+	})
+	hosts, err := Find(query)
+	if err != nil {
+		return nil, errors.Wrap(err, "Error finding containers")
+	}
+
+	return hosts, nil
+}
+
+// find all containers belonging to parent of current host
+func (h *Host) FindHostContainers() ([]Host, error) {
+	if !h.IsParent {
+		return nil, errors.New("Host is not a parent of any containers")
+	}
+	query := db.Query(bson.M{
+		ParentID: h.ParentID,
+	})
+	hosts, err := Find(query)
+	if err != nil {
+		return nil, errors.Wrap(err, "Error finding containers")
+	}
+
+	return hosts, nil
+}
+
+// find parent of container
+func (h *Host) FindParentOfContainer() (*Host, error) {
```

Comment:
Given that we can find hosts by ID, FindOneId(h.ParentID) basically covers most of this. Should we just use that instead.

Reply:
doesn't `FindOneId` find by `_id`, while I want to find by `parent_id`?

Reply:
done

Reply:
GetParent or GetContainerParent()?